pyccel/codegen/printing/fcode.py

```diff
@@ -33,7 +33,7 @@
 from pyccel.ast.variable  import (Variable,
                              IndexedElement,
                              InhomogeneousTupleVariable,
-                             DottedName, PyccelArraySize)
+                             DottedName, NumpyArraySize)
```

Comment:
Again `NumpyArraySize` is imported from the wrong module! This class was defined in `pyccel.ast.internals`.